examples/Demo_Eigendistortion.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<p>maybe improve the alpha values? the distortions are pretty hard to see.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LabForComputationalVision/plenoptic/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>done. </p>